### PR TITLE
Dịch "fashion-mnist_vn" - Phần 1 

### LINKED ISSUE #756: Dịch "fashion-mnist_vn" - Phần 1

### PR DESCRIPTION
**Pull request by quangnhat185**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #756 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 1]
```diff
@@ -32,15 +36,17 @@ d2l.use_svg_display()
...
+## Tải về Bộ dữ liệu
 
 <!--
 Just as with MNIST, Gluon makes it easy to download and load the FashionMNIST dataset into memory via the `FashionMNIST` class contained in `gluon.data.vision`.
 We briefly work through the mechanics of loading and exploring the dataset below.
 Please refer to :numref:`sec_naive_bayes` for more details on loading data.
 -->
 
-*dịch đoạn phía trên*
+Cũng chúng như MNIST, Gluons giúp việc tải và nạp bộ dữ liệu FashionMNIST vào bộ nhớ với class `FashionMNIST` tại `gluon.data.vision` trở nên dễ dàng.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Cũng giống như với MNIST, Gluons giúp việc tải và nạp bộ dữ liệu FashionMNIST vào bộ nhớ với lớp `FashionMNIST` tại `gluon.data.vision` trở nên dễ dàng. 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 2]
```diff
@@ -52,7 +58,8 @@ FashionMNIST consists of images from 10 categories, each represented by 6k image
 Consequently the training set and the test set contain 60k and 10k images, respectively.
 -->
 
-*dịch đoạn phía trên*
+FashionMNIST chứa các hình ảnh từ 10 danh mục, mỗi danh mục được thể hiện bằng 6 nghìn ảnh trong tập huấn luyện và 1 ngìn ảnh trong tập kiểm tra.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
FashionMNIST chứa các hình ảnh từ 10 danh mục, mỗi danh mục được thể hiện bằng sáu nghìn tấm ảnh trong tập huấn luyện và một nghìn tấm ảnh trong tập kiểm tra. 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 3]
```diff
@@ -52,7 +58,8 @@ FashionMNIST consists of images from 10 categories, each represented by 6k image
 Consequently the training set and the test set contain 60k and 10k images, respectively.
 -->
 
-*dịch đoạn phía trên*
+FashionMNIST chứa các hình ảnh từ 10 danh mục, mỗi danh mục được thể hiện bằng 6 nghìn ảnh trong tập huấn luyện và 1 ngìn ảnh trong tập kiểm tra. 
+Do đó tập huấn luyện và tập kiểm tra sẽ chứa tổng lần lượt 60 nghìn và 10 nghìn ảnh.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Do đó, tập huấn luyện và tập kiểm tra sẽ chứa tổng lần lượt 60 nghìn và 10 nghìn tấm ảnh.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 4]
```diff
@@ -17,7 +17,11 @@ To up the ante just a bit, we will focus our discussion in the coming sections o
 but comparatively complex Fashion-MNIST dataset :cite:`Xiao.Rasul.Vollgraf.2017`, which was released in 2017.
 -->
 
-*dịch đoạn phía trên*
+Ở :numref:`sec_naive_bayes`, chúng ta đã huấn luyện bộ phân loại Naive Bayes, sử dụng bộ dữ liệu MNIST được giới thiệu vào năm 1998 :cite:`LeCun.Bottou.Bengio.ea.1998`.
+Mặc dù MNIST hoạt động tốt như bộ dữ liệu đánh giá xếp hạng (_benchmark_), các mô hình đơn giản dựa trên tiêu chuẩn ngày nay cũng có thể đạt được độ chính xác phân loại lên tới 95%.
+Điều này khiến cho việc phân biệt độ mạnh yếu của các mô hình mạnh trở nên không chắc chắn. 
+Ngày nay, MNIST đóng vai trò như là một bài kiểm tra tính năng hơn là bài kiểm tra đánh giá xếp hạng.   
+Để cải thiện vấn đề này, chúng ta sẽ cùng tập trung bàn luận ở các mục tiếp theo về một bộ dữ liệu tương tự nhưng phức tạp hơn, đó là bộ dữ liệu Fashion-MNIST  :cite:`Xiao.Rasul.Vollgraf.2017` được giới thiệu vào năm 2017.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Để cải thiện vấn đề này, chúng ta sẽ cùng tập trung thảo luận ở các mục tiếp theo về một bộ dữ liệu tương tự nhưng phức tạp hơn, đó là bộ dữ liệu Fashion-MNIST  :cite:`Xiao.Rasul.Vollgraf.2017` được giới thiệu vào năm 2017.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 5]
```diff
@@ -63,7 +70,8 @@ The images in Fashion-MNIST are associated with the following categories: t-shir
 The following function converts between numeric label indices and their names in text.
 -->
 
-*dịch đoạn phía trên*
+Hình ảnh trong Fashion-MNIST tương ứng với các danh mục sau: áo phông, quần dài, áo thun, váy, áo khoác, dép, áo sơ-mi, giày thể thao (_sneaker_), túi và giày cao gót. 
+Hàm dưới đây chuyển đổi các nhãn giá trị số sang tên của chúng dưới dạng văn bản.
```
Comment (duythanhvn) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Hàm dưới đây giúp chuyển đổi các nhãn giá trị số sang tên của chúng dưới dạng văn bản.
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 6]
```diff
@@ -63,7 +70,8 @@ The images in Fashion-MNIST are associated with the following categories: t-shir
 The following function converts between numeric label indices and their names in text.
 -->
 
-*dịch đoạn phía trên*
+Hình ảnh trong Fashion-MNIST tương ứng với các danh mục sau: áo phông, quần dài, áo thun, váy, áo khoác, dép, áo sơ-mi, giày thể thao (_sneaker_), túi và giày cao gót.
```
Comment (lkhphuc) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Hình ảnh trong Fashion-MNIST tương ứng với các danh mục sau: áo phông, quần dài, áo thun, váy, áo khoác, dép, áo sơ-mi, giày thể thao, túi và giày cao gót. 
```
Từ này có cần chú thích ko nhỉ?

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 7]
```diff
@@ -77,7 +85,7 @@ def get_fashion_mnist_labels(labels):
 We can now create a function to visualize these examples.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta có thể tạo một hàm để có thể hình dung được các ví dụ này.
```
Comment (lkhphuc) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Chúng ta có thể tạo một hàm để minh hoạ các ví dụ này. 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 8]
```diff
@@ -32,15 +36,17 @@ d2l.use_svg_display()
...
 
 <!--
 Just as with MNIST, Gluon makes it easy to download and load the FashionMNIST dataset into memory via the `FashionMNIST` class contained in `gluon.data.vision`.
 We briefly work through the mechanics of loading and exploring the dataset below.
 Please refer to :numref:`sec_naive_bayes` for more details on loading data.
 -->
 
-*dịch đoạn phía trên*
+Cũng giống như với MNIST, Gluons giúp việc tải và nạp bộ dữ liệu FashionMNIST vào bộ nhớ với lớp `FashionMNIST` tại `gluon.data.vision` trở nên dễ dàng. 
+Cơ chế của việc nạp và trích xuất bộ dữ liệu được mô tả như dưới đây.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Các cơ chế của việc nạp và khám phá bộ dữ liệu sẽ được hướng dẫn ngắn gọn bên dưới.
```
Cách anh dịch câu này hơi mất nghĩa.

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 9]
```diff
@@ -52,7 +58,8 @@ FashionMNIST consists of images from 10 categories, each represented by 6k image
 Consequently the training set and the test set contain 60k and 10k images, respectively.
 -->
 
-*dịch đoạn phía trên*
+FashionMNIST chứa các hình ảnh từ 10 danh mục, mỗi danh mục được thể hiện bằng sáu nghìn tấm ảnh trong tập huấn luyện và một nghìn tấm ảnh trong tập kiểm tra.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
FashionMNIST chứa các hình ảnh từ 10 lớp, mỗi danh mục được thể hiện bằng sáu nghìn tấm ảnh trong tập huấn luyện và một nghìn tấm ảnh trong tập kiểm tra. 
```

Trong bài toán đa lớp thì nhóm thống nhất dịch `category` là `lớp` để đỡ bối rối khi đọc

File: chapter_linear-networks/fashion-mnist_vn.md (hunk 6 above)
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Hình ảnh trong Fashion-MNIST tương ứng với các lớp sau: áo phông, quần dài, áo thun, váy, áo khoác, dép, áo sơ-mi, giày thể thao (_sneaker_), túi và giày cao gót. 
```

File: chapter_linear-networks/fashion-mnist_vn.md (hunk 7 above)
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Chúng ta có thể tạo một hàm để có thể hình dung được các mẫu này. 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 10]
```diff
@@ -99,7 +107,7 @@ def show_images(imgs, num_rows, num_cols, titles=None, scale=1.5):
 Here are the images and their corresponding labels (in text) for the first few examples in the training dataset.
 -->
 
-*dịch đoạn phía trên*
+Dưới đây là các hình ảnh và nhãn tương ứng của chúng (ở dạng văn bản) từ một vài ví dụ đầu tiên trong tập huấn luyện.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Dưới đây là các hình ảnh và nhãn tương ứng của chúng (ở dạng văn bản) từ một vài mẫu đầu tiên trong tập huấn luyện. 
```

Mình dịch là `ví dụ` em thấy có thể gây hiểu lầm

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 11]
```diff
@@ -17,7 +17,11 @@ To up the ante just a bit, we will focus our discussion in the coming sections o
 but comparatively complex Fashion-MNIST dataset :cite:`Xiao.Rasul.Vollgraf.2017`, which was released in 2017.
 -->
 
-*dịch đoạn phía trên*
+Ở :numref:`sec_naive_bayes`, chúng ta đã huấn luyện bộ phân loại Naive Bayes, sử dụng bộ dữ liệu MNIST được giới thiệu vào năm 1998 :cite:`LeCun.Bottou.Bengio.ea.1998`.
+Mặc dù MNIST hoạt động tốt như bộ dữ liệu đánh giá xếp hạng (_benchmark_), các mô hình đơn giản dựa trên tiêu chuẩn ngày nay cũng có thể đạt được độ chính xác phân loại lên tới 95%.
+Điều này khiến cho việc phân biệt độ mạnh yếu của các mô hình mạnh trở nên không chắc chắn. 
+Ngày nay, MNIST đóng vai trò như là một bài kiểm tra tính năng hơn là bài kiểm tra đánh giá xếp hạng.
```
Comment (aivivn) [inline chapter_linear-networks/fashion-mnist_vn.md]: `sanity check` dịch là gì được nhỉ? Từ này trong ngành dùng nhiều.

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 12]
```diff
@@ -77,7 +85,7 @@ def get_fashion_mnist_labels(labels):
 We can now create a function to visualize these examples.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta có thể tạo một hàm để minh hoạ các ví dụ này.
```
Comment (rootonchair) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Chúng ta có thể tạo một hàm để minh hoạ các mẫu này. 
```

File: chapter_linear-networks/fashion-mnist_vn.md (hunk 11 above)
Comment (tiepvupsu) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Ngày nay, MNIST đóng vai trò như là một bài kiểm tra sơ bộ hơn là bài kiểm tra đánh giá xếp hạng.   
```

Comment (quangnhat185) [reply chapter_linear-networks/fashion-mnist_vn.md]: > `sanity check` dịch là gì được nhỉ? Từ này trong ngành dùng nhiều.
Lão đại @tiepvupsu dịch là `sơ bộ` mình thấy khá ổn trong ngữ cảnh này

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 13]
```diff
@@ -290,7 +298,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 
 * Đoàn Võ Duy Thanh
 <!-- Phần 1 -->
-*
+* Nguyễn Lê Quang Nhật
```
Comment (quangnhat185) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
* Nguyễn Lê Quang Nhật
* @tiepvupsu 
* @lkhphuc 
* @rootonchair 
* @duythanhvn 
```

File: chapter_linear-networks/fashion-mnist_vn.md [hunk 14]
```diff
@@ -17,7 +17,11 @@ To up the ante just a bit, we will focus our discussion in the coming sections o
 but comparatively complex Fashion-MNIST dataset :cite:`Xiao.Rasul.Vollgraf.2017`, which was released in 2017.
 -->
 
-*dịch đoạn phía trên*
+Ở :numref:`sec_naive_bayes`, chúng ta đã huấn luyện bộ phân loại Naive Bayes, sử dụng bộ dữ liệu MNIST được giới thiệu vào năm 1998 :cite:`LeCun.Bottou.Bengio.ea.1998`.
+Mặc dù MNIST hoạt động tốt như bộ dữ liệu đánh giá xếp hạng (_benchmark_), các mô hình đơn giản dựa trên tiêu chuẩn ngày nay cũng có thể đạt được độ chính xác phân loại lên tới 95%.
+Điều này khiến cho việc phân biệt độ mạnh yếu của các mô hình mạnh trở nên không chắc chắn.
```
Comment (quangnhat185) [inline chapter_linear-networks/fashion-mnist_vn.md]:
```suggestion
Điều này khiến cho việc phân biệt độ mạnh yếu của các mô hình trở nên không chắc chắn. 
```
